.github/workflows/lint.yml

```
@@ -0,0 +1,52 @@
+
+permissions:
+  contents: read
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)